app/(app)/session/[session].tsx

```diff
@@ -1,40 +1,184 @@
+                          {showMoreBio ? ' ...Show less' : ' Show more'}
+                        </StyledText>
+                      )}
+                    </StyledText>
+                  </StyledText>
+                  <Space size={10} />
+                </View>
+              ))
+            ) : (
+              <StyledText font="regular" style={{ color: colors.textLight }}>
+                {!showMoreBio ? truncate(140, session?.speakers[0]?.biography) : session?.speakers[0]?.biography}
+                {session?.speakers[0]?.biography && session?.speakers[0]?.biography.length > 140 && (
+                  <StyledText
+                    size="sm"
+                    font="semiBold"
+                    style={{ color: colors.primary }}
+                    onPress={() => setShowMoreBio(!showMoreBio)}
+                  >
+                    {showMoreBio ? ' ...Show less' : ' Show more'}
+                  </StyledText>
+                )}
+              </StyledText>
+            )}
+
+            <Space size={16} />
+
+            <Image
+              source={{ uri: session?.session_image || '' }}
+              style={styles.image}
+              contentFit="fill"
+              contentPosition="left center"
+            />
+          </View>
+
+          <View style={[styles.centered, { borderColor: dark ? colors.background : colors.border }]}>
+            <StyledText font="light">{getSessionTimesAndLocation(session?.slug || '')}</StyledText>
+
+            <Space size={16} />
+
+            <View style={[styles.chip, { backgroundColor: dark ? colors.background : colors.text }]}>
+              <StyledText style={[styles.chipText, { color: dark ? colors.text : colors.background }]}>
+                #{session?.session_level}
+              </StyledText>
+            </View>
+          </View>
+
+          <View style={styles.withPadding}>
+            {session?.speakers && session?.speakers.length > 1 ? (
+              <View>
+                <StyledText font="medium" size="lg">
+                  Twitter Handles
+                </StyledText>
+
+                <Space size={16} />
+
+                <Row>
+                  {session?.speakers.map((speaker, index) => (
+                    <Pressable
+                      key={index.toString()}
+                      style={[styles.button, { borderColor: colors.primary, backgroundColor: colors.background }]}
+                    >
+                      <FontAwesome5 name="twitter" size={20} color={colors.primary} />
+                      <Space size={4} horizontal />
+                      <StyledText font="medium" style={{ color: colors.primary }}>
+                        {speaker.twitter ? getTwitterHandle(speaker.twitter) : 'N/A'}
+                      </StyledText>
+                    </Pressable>
+                  ))}
+                </Row>
+              </View>
+            ) : (
+              <Row>
+                <StyledText font="medium" size="lg">
```

Comment:
text size for this label should be "md" as per the design. 
"md" is already default so when you remove "lg" it will be fine.